libs/payments/ui/src/lib/utils/types.ts

```diff
@@ -1,6 +1,12 @@
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+export enum ButtonVariant {
```

Comment:
Since this is only relevant to the `BaseButton` component, it should be included there.